File: Music/git_project2/codeforces/199A.py
input1 = int(raw_input())
first = 0
second = 0
temp = 0
newFib = 1
while newFib < input1:
	first = second
	temp = newFib
	newFib = newFib + second
	second = temp
if input1 == 1:
	second = 1
print second - first + " " + first + " " + first
	

File: Music/git_project2/codeforces/199A.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
	long long int input,first=0, second=0,temp=0,newFib = 1;
	cin>>input;
	while (newFib<input)
	{
		first = second;
		temp = newFib;
		newFib += second;
		second = temp;
	}
	if(input == 1)
	second = 1;
	cout<<second - first<<" "<<first<<" "<<first;
	return 0;
}
